src/pages/Index/index.tsx

```diff
@@ -1,27 +1,40 @@
 import * as React from 'react';
-import { Link } from 'react-router-dom';
 import { Col } from 'react-bootstrap';
 
-type PublicProps = {};
+import { gettext } from '../../utils';
 
-export class IndexBase extends React.Component<PublicProps> {
+type Props = {};
+
+export class IndexBase extends React.Component<Props> {
   render() {
+    const apiHost = process.env.REACT_APP_API_HOST;
+    const repoUrl = 'https://github.com/mozilla/addons-code-manager';
+
     return (
       <Col>
         <p>
-          There is nothing you can do here, but try{' '}
```

Comment:
You could remove the *nothing to do here* but I would still like to see the links to the add-ons below, shown only on dev. It's hard to find examples on dev of add-ons with a big file tree and global / inline linter messages of varying types. The first page of dev today only has one extension with global linter messages and none with inline linter messages:

<img width="865" alt="Screenshot 2019-04-22 10 03 46" src="https://user-images.githubusercontent.com/55398/56507236-3ff83e80-64e6-11e9-903a-58276d3d5109.png">

Reply:
@kumar303 where do we want to see the links? Local dev? Local dev and (hosted) -dev?